v03_pipeline/lib/model/dataset_type.py

```diff
@@ -165,7 +167,13 @@ def formatting_annotation_fns(self) -> list[Callable[..., hl.Expression]]:
                 shared.variant_id,
                 shared.xpos,
             ],
-            DatasetType.MITO: [
+            (ReferenceGenome.GRCh37, DatasetType.SNV_INDEL): [
```

Comment:
similar to my above comment, I think it would be better to have a constant for the shared fields so it would be more explicit what differs between builds